sdk/src/connections/target/adsd3500_sensor.cpp

```diff
@@ -118,7 +118,8 @@ Adsd3500Sensor::Adsd3500Sensor(const std::string &driverPath,
     : m_driverPath(driverPath), m_driverSubPath(driverSubPath),
       m_captureDev(captureDev), m_implData(new Adsd3500Sensor::ImplData),
       m_firstRun(true), m_adsd3500Queried(false), m_depthComputeOnTarget(true),
-      m_chipStatus(0), m_imagerStatus(0) {
+      m_chipStatus(0), m_imagerStatus(0),
+      m_hostConnectionType(aditof::ConnectionType::NETWORK) {
```

Comment:
Why are you setting it to NETWORK? (and not ON_TARGET?)

Reply:
We can set it to ON_TARGET as well by default. The idea would be to have it at least once initialized.

Reply:
ON_TARGET would make more sense

Reply:
I agree with it, is there a reason why?

Reply:
An object of type Adsd300Sensor class will only be instantiated on target. So it makes sense to set, in its constructor, the connection type to reflect this. Otherwise, when running SDK on target, CameraDetails will say the connection type is network confusing users.
There's the NetworkSensor class that will set the connection type to network.